server/src/main/java/org/elasticsearch/http/HttpHandlingSettings.java

```diff
@@ -71,54 +39,7 @@ public static HttpHandlingSettings fromSettings(Settings settings) {
             SETTING_HTTP_RESET_COOKIES.get(settings),
             SETTING_HTTP_COMPRESSION.get(settings),
             SETTING_HTTP_COMPRESSION_LEVEL.get(settings),
-            SETTING_HTTP_DETAILED_ERRORS_ENABLED.get(settings),
-            SETTING_PIPELINING_MAX_EVENTS.get(settings),
```

Comment:
Why are these removed?

Reply:
These aren't used from this object, we load them into fields on the actual http transport and use them there.